test/unit/test_manifest.py

```diff
@@ -239,7 +239,7 @@ def test__no_nodes(self):
                 'parent_map': {},
                 'child_map': {},
                 'metadata': {
-                    'generated_at': '2018-02-14T09:15:13Z',
+                    'generated_at': '2018-02-14T09:15:13',
```

Comment:
That's a good question. Hologram had a datetime encoder which in some cases would just append a 'Z' to the serialized output, which was not a great way to do that. It wasn't totally clear to me when that would happen, so we should crosscheck with previous artifact metadata.

Reply:
I am not an expert here, but I do think that the inclusion of the timezone (Z = UTC) is necessary for the timezone to be ISO 8601 compliant. If i recall correctly, we had some issues when trying to render timestamps on the web (eg. dbt-docs, dbt Cloud) when these timestamps _lacked_ a timezone and were not ISO 8601 compliant!

Reply:
If we have times that don't have a timezone, I suspect it's probably because we're not using the right function (utc_now as opposed to now). Just appending a 'Z' is not the correct way to do it, though. Will see if I can identify where that might be happening.

Reply:
So apparently the Python datetime class does not return times with the 'Z', but just concatenating Z on the end is kind of wrong too, i.e. it won't always result in a correct time string. Will look for a solution.

Reply:
So I guess concatenating the Z is what we need to do. Could also explicitly add a 00:00 timezone thing, but that's hardly better. I'm going to create an encoder feature for Mashumaro to handle this.

Reply:
This has been fixed.